scripts/speech_recognition/convert_to_tarred_audio_dataset.py

```diff
@@ -569,6 +582,25 @@ def _read_manifest(self, manifest_path: str, config: ASRTarredDatasetConfig):
 
         return entries, total_duration, filtered_entries, filtered_duration
 
+    def _write_to_tar(self, tar, audio_filepath: str, squashed_filename: str) -> None:
+        if (codec := self.config.force_codec) is None or audio_filepath.endswith(f".{codec}"):
```

Comment:
This is python 3.10 syntax right ? Let's try to keep it runnable on lower python versions. Min version is 3.10 but mostly asr continues to work on 3.8 or so

Reply:
It's python 3.8, we should be fine 

https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions

Reply:
Perfect